subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -572,6 +572,20 @@ void ll_cis_create(uint16_t cis_handle, uint16_t acl_handle)
 	cis->lll.datapath_ready_rx = 0;
 	cis->lll.tx.bn_curr = 1U;
 	cis->lll.rx.bn_curr = 1U;
+	/* Set the flush points in nr. of subevents */
+	/* TODO: verify that the use of DIV_ROUND_UP is the right thing to do */
+	if (cis->lll.tx.bn != 0) {
+		cis->lll.tx.ft_cntr_se = cis->lll.tx.ft *
+			DIV_ROUND_UP(cis->lll.nse, cis->lll.tx.bn);
+	} else {
+		cis->lll.tx.ft_cntr_se = 0;
```

Comment:
Would assigning to `1U` make any difference. I prefer `1U` if there is no mention that it shall be 0 when bn is 0.